docs/maui/DeveloperGuide.md

```diff
@@ -1,88 +1,101 @@
 
-## <a name="snippets">Code snippets</a> ##
+## <a name="code_snippets">Code Snippets</a> ##
 
 ### <a name="configure_authenticator">Configure MobileAuthenticator</a> ###
 
-Create a single instance of the MobileAuthenticator which will remain for the lifetime of the application. The registration of this object requiring dependency injection should be performed in the CreateMauiApp method of MauiProgram class.
+Create a single instance of the MobileAuthenticator which will remain for the lifetime of the application. The registration of this object requiring dependency injection should be performed in the `CreateMauiApp` method of MauiProgram class.
 
-```
-builder.Services.AddSingleton(new MobileAuthenticator
-    (
-      <"ENDPOINT_PROVIDER">, // OpenIdEndpointProvider.Staging or OpenIdEndpointProvider.Production
-      <"CLIENT_ID">,
-      new[]{<"SCOPES">},
-      <"REDIRECT_URI">", // REDIRECT_URI added in Trimble cloud console.
-      allowPersistenceStorage:true
-    )
+```csharp
+var options = new MobileAuthenticatorOptions
+  {
+    EndpointProvider = "<ENDPOINT_PROVIDER>", // OpenIdEndpointProvider.Staging or OpenIdEndpointProvider.Production
+    ClientId = "<CLIENT_ID>",
+    Scopes = new[] { "<SCOPES>" },
+#if WINDOWS
+    RedirectUri = "<LOCALHOST_REDIRECT_URI>",
+#else
+    RedirectUri = "<REDIRECT_URI>",
+#endif
+    EnableTokenPersistence = true
+  };
+
+builder.Services.AddSingleton(new MobileAuthenticator(options))
 ```
 
 ### <a name="set_currentview">Set the current view to launch browser</a> ###
 
-Once the MobileAuthenticator instance is created, we must pass the Activity/ViewController that launches browser during login/logout to the MobileAuthenticator instance. To achieve this the following platform-specific method calls are required:
+Once the MobileAuthenticator instance is created, we must pass the Activity/ViewController in Android/iOS that launches browser during login/logout to the MobileAuthenticator instance. To achieve this the following platform-specific method calls are required:
 
-### iOS
+<b>Android</b>
 
-The viewController that launches the browser during login must be passed using the WithViewController method.
+Activity that launches the browser during login must be passed using the `WithActivity` method.
 
-```
-mobileAuthenticator.WithViewController(viewController);
+```csharp
+mobileAuthenticator.WithActivity(activity);
 ```
 
-### Android
+<b>iOS</b>
 
-Activity that launches the browser during login must be passed using the WithActivity method.
+The viewController that launches the browser during login must be passed using the `WithViewController` method.
 
+```csharp
+mobileAuthenticator.WithViewController(viewController);
 ```
-mobileAuthenticator.WithActivity(activity);
-```
+
+<b>Windows</b>
+
```

Comment:
@vasudev-j mention about localhost redirection here, give descriptive message how this is being done in our SDK

Reply:
Added description about localhost redirection in 'Handling the redirect' section.